db/migrate/20200415145622_add_task_to_user.rb

```diff
@@ -0,0 +1,7 @@
+class AddTaskToUser < ActiveRecord::Migration[6.0]
+  def change
+    change_table :tasks do |t|
+      t.belongs_to :user, null: false, foreign_key: true
```

Comment:
What exactly is this doing? 
Is it adding a task to user table?
or user id to task table??

Reply:
Task will belong to user, 

User_id in Task table.